returnn/datasets/basic.py

```diff
@@ -403,8 +403,8 @@ def get_seq_order_for_epoch(self, epoch, num_seqs, get_seq_len=None):
       nth = int(tmp[1]) if len(tmp) > 1 else 1
       # Keep this deterministic! Use fixed seed.
       rnd_seed = self._get_random_seed_for_epoch(epoch=epoch, num_epochs_fixed=nth)
-      numpy.random.seed(rnd_seed)
-      seq_index = numpy.random.permutation(num_seqs)
+      random_generator = numpy.random.default_rng(rnd_seed)
```

Comment:
This is now the recommended way and works with numpy >= 1.17. Ok or rather use `numpy.random.RandomState`?

Reply:
Ok, done.